File: account-manage/account-manage/src/main/java/com/service/account_manage/entity/id/AccountCodeId.java
package com.service.account_manage.entity.id;

import jakarta.persistence.Column;
import jakarta.persistence.Embeddable;
import lombok.*;

import java.io.Serializable;
import java.util.Objects;

@Embeddable
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
@EqualsAndHashCode
public class AccountCodeId implements Serializable {

    private String userId;

    private String codeId;

    private String comCd;
}

File: app-server-account-manage/account_manage/src/main/kotlin/com/service/account_manage/entity/id/AccountCodeId.kt
package com.service.account_manage.entity.id

import jakarta.persistence.Column
import jakarta.persistence.Embeddable
import java.io.Serializable

@Embeddable
data class AccountCodeId(
    @Column(name = "USER_ID", length = 20, nullable = false)
    val userId : String?= null,

    @Column(name = "CODE_ID", length = 20)
    val codeId : String?= null
):Serializable
